src/execution_plan/execution_plan.h

```diff
@@ -23,6 +23,8 @@ struct ExecutionPlan {
 	// NOTE - segments and segment_count are only stored for proper freeing.
 	int segment_count;                  // Number of ExecutionPlan segments.
 	ExecutionPlan **segments;           // Partial execution plans scoped to a subset of operations.
+	// Semi-independent sub execution plans which created during the build of the main execution plan.
+	ExecutionPlan **sub_execution_plans;
```

Comment:
The distinction between these arrays is confusing, but I guess there's not much to be done about that!